test/basic-querying.test.js

```diff
@@ -23,6 +23,7 @@ describe('basic-querying', function() {
       birthday: {type: Date, index: true},
       role: {type: String, index: true},
       order: {type: Number, index: true, sort: true},
+      notes: {type: String, index: true},
```

Comment:
@mitsos1os Wouldn't the property here be `tag` instead of `notes`? The connector's tests fail due to `tag` is not defined in the model(therefore no mapping column in the mysql/mssql schema).

Reply:
Fixed!